crypto/key_manager.go

```diff
@@ -84,8 +79,8 @@ func parsePrivateKey(key []byte) (crypto.PrivateKey, error) {
 }
 
 // NewFromPrivatePEM returns key manager for a password protected PEM object.
```

Comment:
The comment here needs changing to make it obvious that the PEM doesn't *need* to be password-protected.